includes/class-sensei-usage-tracking-data.php

```diff
@@ -51,7 +52,111 @@ public static function get_usage_data() {
 			'teachers'                => self::get_teacher_count(),
 		);
 
-		return array_merge( $question_type_count, $usage_data );
+		return array_merge( $question_type_count, $usage_data, $quiz_stats );
+	}
+
+	/**
+	 * Get stats related to lesson quizzes.
+	 *
+	 * @since 1.11.0
+	 *
+	 * @return array
+	 */
+	private static function get_quiz_stats() {
+		$query = new WP_Query( array(
+			'post_type'      => 'lesson',
+			'fields'         => 'ids',
+			'posts_per_page' => -1,
+			'no_found_rows'  => true,
+			'meta_query'     => array(
+				array(
+					'key'   => '_quiz_has_questions',
+					'value' => true,
+				),
+				array(
+					'key'     => '_lesson_course',
+					'value'   => '',
+					'compare' => '!=',
+				),
+				array(
+					'key'     => '_lesson_course',
+					'value'   => '0',
+					'compare' => '!=',
+				),
+			),
+		) );
+
+		$stats = array(
+			'quiz_total'         => 0,
+			'questions_min'      => null,
+			'questions_max'      => null,
+			'category_questions' => 0,
+		);
+		$question_counts    = array();
+		$published_quiz_ids = array();
+
+		foreach ( $query->posts as $lesson_id ) {
+			$course_id = Sensei()->lesson->get_course_id( $lesson_id );
+			if ( empty( $course_id ) || 'publish' !== get_post_status( $lesson_id ) || 'publish' !== get_post_status( $course_id ) ) {
+				continue;
+			}
+
+			$quiz_id              = Sensei()->lesson->lesson_quizzes( $lesson_id );
+			$quiz_question_posts  = Sensei()->lesson->lesson_quiz_questions( $quiz_id );
+			$question_counts[]    = count( $quiz_question_posts );
+			$published_quiz_ids[] = $quiz_id;
+			$stats['quiz_total']++;
+		}
+
+		if ( ! empty( $published_quiz_ids ) ) {
+			$multiple_question_query     = new WP_Query( array(
```

Comment:
I don't think this is needed here.

Reply:
You're right! I moved it to its own method and forgot to remove the code. Done!